oat-sa/extension-tao-testqti-previewer
# Feature/nex 19 adapt the test item previewer

Related to https://oat-sa.atlassian.net/browse/NEX-19

Adapt the test/item previewer to the new initialization format.
Migrate the extension to ES2015 by the way.

Requires : 
 - [x] https://github.com/oat-sa/extension-tao-test/pull/289

How to test : 
- View some results
- Click the "review" button, it should open a test runner to preview an item

Run 
```
cd tao/views/build
npx grunt connect:test taoqtitestpreviewertest
```
